src/pages/Rockets.js

```diff
@@ -1,3 +1,33 @@
-const Rockets = () => (<div>Rockets</div>);
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchRocketsFromServer } from '../redux/rockets/rockets';
+
+const Rockets = () => {
+  const rockets = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!rockets.length) dispatch(fetchRocketsFromServer());
+  }, []);
+
+  return (
+    <div className="rocket">
+      {rockets.map((rocket) => {
+        const {
+          id, name, type, images,
+        } = rocket;
+
+        return (
+          <div key={id} className="eachRocket">
+            <img src={images} alt="rocket" className="rocket-image" />
+            <p>{name}</p>
+            <p>{type}</p>
+          </div>
+        );
+      })}
+      ;
```

Comment:
It seems by mistake this has remained in the file. I think it needs to be removed. :bangbang:

Reply:
Consider it Done 👍 :